src/models/SimplificationOptions.ts

```diff
@@ -0,0 +1,7 @@
+
+/**
+ * Options passed along in the simplification stage.
+ */
+export class SimplificationOptions {
```

Comment:
I don't think so that it should be as class. Please change to interface.

Reply:
Then I cant have default values? 🤔

Reply:
Yes, in Simplify class's constructor.

Reply:
```ts
class Simplifier {
  private static defaultOptions: SimplificationOptions = {}

  constructor(
    options: SimplificationOptions = Simplifier.defaultOptions,
  ) {
    this.options = { ...Simplifier.defaultOptions, ...options }
  }
}
```